runtime/j9vm/jvm.c

```diff
@@ -1514,6 +1524,88 @@ static jint initializeReflectionGlobals(JNIEnv * env, BOOLEAN includeAccessors)
  *	DLL: jvm
  */
 jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args) {
+	return JNI_CreateJavaVM_impl(pvm, penv, vm_args, FALSE);
+}
+
+#if defined(JITSERVER_SUPPORT)
+int32_t JNICALL 
+JITServer_CreateServer(JITServer **jitServer, void *vm_args)
+{
+	JNIEnv *env = NULL;
+	JITServer *server = NULL;
+	jint rc = JITSERVER_OK;
+
+	server = malloc(sizeof(JITServer));
+	if (NULL == server) {
+		rc = JITSERVER_OOM;
+		goto _end;
+	}
+	server->startJITServer = startJITServer;
+	server->waitForJITServerTermination = waitForJITServerTermination;
+	rc = JNI_CreateJavaVM_impl(&server->jvm, (void **)&env, vm_args, TRUE);
```

Comment:
Is there existing check to prevent multiple JVMs from being created in the same process?

As for the tests, I think [`vm_lifecycle`](https://github.com/eclipse/openj9/tree/master/runtime/tests/vm_lifecycle) tests seem to be appropriate place for doing these checks, however, currently they are disabled https://github.com/eclipse/openj9/blob/ecb16ff5c4fc581484f6c9acaa5e0891082c4423/test/functional/NativeTest/playlist.xml#L531.

I guess this PR can be merged without these checks for now. If that is ok, then I will address the checks in a separate issue, as it also needs the `vm_lifecycle` tests to be enabled again.

Reply:
Personally I don't think we need to add any check. It's the user responsability to not misuse an API.

Reply:
The check occurs in the public JNI_CreateJavaVM() API in jvm.c using a static variable, the JNI_CreateJavaVM() in this file is internal.
```
	if (BFUjavaVM != NULL) {
		result = JNI_ERR;
		goto exit;
	}
```

Reply:
Sorry, got confused about which file I was looking at. Anyway, JITServer_CreateServer() should add the above check.